test/sagemaker_tests/pytorch/training/integration/sagemaker/test_dgl.py

```diff
@@ -41,6 +45,12 @@ def test_dgl_gcn_training_cpu(sagemaker_session, ecr_image, instance_type):
 @pytest.mark.skip_cpu
 @pytest.mark.skip_py2_containers
 def test_dgl_gcn_training_gpu(sagemaker_session, ecr_image, instance_type):
+
+    _, image_framework_version = get_framework_and_version_from_tag(ecr_image)
+    image_cuda_version = get_cuda_version_from_tag(ecr_image)
+    if Version(image_framework_version) == Version("1.6") and image_cuda_version == "cu110":
+        pytest.skip("DGL does not suport CUDA 11 for PyTorch 1.6")
```

Comment:
Does this test still need to be skipped? DGL has been installed using a custom-built wheel, right?

Reply:
Even with custom build this issue is noted